packages/babel-cli/src/babel/dir.js

```diff
@@ -43,18 +43,24 @@ module.exports = function (commander, filenames) {
     }
   }
 
+  let filterDotFiles = function (x) {
+    return x[0] === ".";
+  };
+  let copyFilesFilter = () => false;
+
   function handle(filename) {
     if (!fs.existsSync(filename)) return;
 
     let stat = fs.statSync(filename);
-
     if (stat.isDirectory(filename)) {
       let dirname = filename;
-
-      _.each(util.readdir(dirname), function (filename) {
+      if (commander.copyDotFiles) {
+        copyFilesFilter = filterDotFiles;
```

Comment:
Since it excludes by _default_, we need to use our custom filter (aka no filter) when `commander.copyDotFiles` _is_ set. This will do the opposite?

(sorry if I wasn't clear in my earlier [comment](https://github.com/babel/babel/pull/5083#discussion_r95397145))